api/src/main/java/me/lokka30/treasury/api/economy/account/PlayerAccount.java

```diff
@@ -172,7 +178,7 @@ default CompletableFuture<Response<BigDecimal>> resetBalance(
                 .newBuilder()
                 .withCurrency(currency)
                 .withInitiator(initiator)
-                .withTransactionAmount(currency.getStartingBalance(getUniqueId()))
+                .withTransactionAmount(this.getStartingBalance(currency))
```

Comment:
```suggestion
                .withTransactionAmount(getStartingBalance(currency))
```

Reply:
Why?

Reply:
I reiterate the same question for prefixing the method with `this`. It's clearly calling the `getStartingBalance` method within itself.

Reply:
I just realized we can remove the method from PlayerAccount and transfer it to just Account. `this.` is a Java feature that is to better illustrate which method/field _exactly_ you are calling.

Reply:
Oops, yes, it was meant to be in Account. Can't believe I missed that in my initial review. Regardless, I don't care much about using `this` to die on a hill lol, but I don't think it's necessary at all here.